web/cobrands/bromley/layout.scss

```diff
@@ -1,135 +1,270 @@
+// because of FMS's selectors being quite specific
+#main-nav ul#main-menu li span {
+  color: $nav_fg;
+  background-color: $nav_fg_hover;
+}
+
+// Override the "report a problem" button because it's not really a button
+#main-nav ul#main-menu li a.report-a-problem-btn {
+  color: $nav_fg;
+}
+
+// Fix bad margin on local alerts form
+body.alertindex form.full-width {
+  margin-left: 0;
 }
 
-// Don't want FixMyStreet logo on desktop, and we have a skip to content in
-// Bromley header
-#site-header {
-    display: none;
+// Override the little banner you get when you're logged in on the map page
+// because FMS makes it a different colour and moves it over to the left.
+body.mappage #user-meta p {
+  background-color: $bromley_blue;
+  border-bottom: 1px solid #444;
+  left: 2.65em;
 }
 
+// And also override the banner on other pages so that it lines up properly
 #user-meta {
-    display: none;
+  max-width: 1200px;
+}
+#user-meta p,
+body.frontpage #user-meta p {
+  top: 0;
+  right: auto;
+  background-color: $bromley_blue;
+  color: #fff;
+  box-shadow: none;
 }
 
-// Currently hiding, but do want some internal navigation somewhere at the end
-.nav-wrapper {
-    display: none;
+// Because we've moved the user-meta, we also have to move the #front-main div
+// down a bit so that it shows the #user-meta
+#front-main {
+  margin-top:3em
+}
+
+// The page h1 needs to be white on desktop, because it's in a blue box
+h1.main {
+  color: #fff;
 }
 
-// White background, so no shadow or margin needed.
-.content {
-    margin: 0;
-    @include box-shadow(none);
+// Because we've changed the page background, the footer looks a bit rubbish
+footer,
+body.twothirdswidthpage .container .content footer .tablewrapper,
+body.fullwidthpage .container .content footer .tablewrapper {
+  background-color: $bromley_blue;
 }
-.ie6, .ie7, .ie8  {
-    .content {
-        margin: 0;
-        border: none;
+
+// Make the powered by FMS logo work in Old IE. FMS fixes this in IE8, but
+// it's broken in IE6 and 7 too.
+.ie6, .ie7, .ie8 {
+  body.twothirdswidthpage,
+  body.fullwidthpage {
+    .container .content footer a.platform-logo {
+      color: #ffffff;
+      background: none;
+      text-indent: 0px;
+      height: auto;
     }
+  }
 }
 
-// As map can scroll and isn't at the top, give it an edge
-#map_box {
-    border: solid 1px #999;
-    top: 1em;
-    right: 0em;
-    margin: 0;
+// This is our custom wrapper for Bromley's header elements
+.bromley-header {
+  // Bromley's logo has a bottom margin to space it out from the A-Z nav, we
+  // can't do that because they're in separate divs, so add a margin here
+  // instead.
+  margin-bottom: 20px;
 }
 
-// Perhaps fix map location (should be in central?)
-.ie6 #map_box, .ie7 #map_box {
-    right: -480px;
+// The reports_list.scss hides the big banner on the sidebar that tells people
+// what to do with the map (click on it). We don't want that, so put it back
```

Comment:
It looks like the report_list assumes a different `click-the-map` class is used (see e.g. Oxfordshire). This should probably be looked at and fixed/consolidated to avoid future confusion.